applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -931,9 +1038,10 @@ static void discover_cb(struct bt_conn *conn, int err, enum bt_audio_dir dir)
 	int ret;
 	uint8_t temp_cap_index;
 
-	struct stream_index idx;
+	struct stream_index idx = {0, 0, 0};
```

Comment:
`discover_cb` is now more than 200 lines long. Consider splitting it into multiple smaller functions.